core/templates/dev/head/pages/collection_editor/editor_tab/CollectionNodeCreatorDirective.js

```diff
@@ -73,6 +90,16 @@ oppia.directive('collectionNodeCreator', [function() {
           );
         };
 
+        var cleanExplorationId = function(newExplorationId) {
```

Comment:
Hmm, OK. I think there's some design considerations here that I'll address once you've handled the merge, but in the meantime please just add a comment in the code to explain what the function is doing. I'll give you a more detailed review once the merge is done.

Thanks!